packages/contracts/test/contracts/OVM/accounts/OVM_ECDSAContractAccount.spec.ts

```diff
@@ -190,4 +190,23 @@ describe('OVM_ECDSAContractAccount', () => {
       )
     })
   })
+  describe('isValidSignature()', () => {
+    // NOTE: There is no good way to unit test verifying a valid signature
+    // An integration test exists testing this instead
+    it(`should revert for a malformed signature`, async () => {
+      const messageHash = utils.keccak256('0x42')
+      await expect(
+        OVM_ECDSAContractAccount.isValidSignature(messageHash, '0xdeadbeef')
+      ).to.be.revertedWith('ECDSA: invalid signature length')
+    })
+    it(`should return 0 for an invalid signature`, async () => {
+      const messageHash = utils.keccak256('0x42')
+      const signature = await wallet.signMessage('0x42')
+      const bytes = await OVM_ECDSAContractAccount.isValidSignature(
+        messageHash,
+        signature
+      )
+      expect(bytes).to.equal('0x00000000')
+    })
+  })
```

Comment:
As a general comment in this test file what's with using `waffle` for the wallet? I have mostly seen Alice and Bob around from `ethers`.

Reply:
I know we've needed to use the waffle wallet to have certain signing functionality that's not available on `ethers.getSigner` signers

Reply:
Tried switching this to an ethers signer and it didn't work